docs/UserGuide.md

```diff
@@ -117,22 +135,77 @@ Examples:
 
 ### Listing assignments : `list`
 
-Format: `list [NUMBER]`
+Format: `list [NUMBER_OF_DAYS]`
 
-- Shows a list of assignments in your schedule within next `NUMBER` 
-days, starting from the current date (and current time).
-- A day represents 24 hours. 
-- `list` without `NUMBER` displays your entire list of assignments 
-stored in ProductiveNUS.
+You can list your assignments altogether with just typing `list`. Alternatively, you can type `list` followed by an index `NUMBER_OF_DAYS` to list your assignments with deadlines that are within the current date (and time) and `NUMBER_OF_DAYS` later.
 
+You can use `NUMBER_OF_DAYS` index to quickly view assignments that you need to complete soon! (with deadlines nearing).
+For example, `list 2` will show you assignments that are due 2 days (48 hours) from the current date (and current time).
+{: .alert .alert-gitlab-purple}
```

Comment:
The phrasing for line 142 sounds a little strange. Maybe "... quickly view assignments that are due soon." would be better?

Reply:
Thought "you need to complete soon" sounds a bit more impactful :')

Reply:
Hmm, then maybe removing the (with deadlines nearing) would be better in terms of phrasing :)